src/__experimental__/components/numeral-date/numeral-date.stories.js

```diff
@@ -97,15 +97,15 @@ export const Validations = () => {
 Basic.story = {
   parameters: {
     chromatic: {
-      disable: false
+      disable: true
     }
   }
 };
 
 Validations.story = {
   parameters: {
     chromatic: {
-      disable: false
+      disable: true
```

Comment:
we can't disable this story, because we have here all rendered `validations` for `Numeral Date` we can provide `parameters={{ chromatic: { disable: true }}}` into `mdx` file and ignore separate validation stories